src/autocomplete.js

```diff
@@ -461,29 +463,32 @@ export default class Autocomplete extends Component {
       'aria-autocomplete': (this.hasAutoselect()) ? 'both' : 'list'
     }
 
-    let dropdownArrow
-
-    // we only need a dropdown arrow if showAllValues is set to a truthy value
-    if (showAllValues) {
-      dropdownArrow = dropdownArrowFactory({ className: dropdownArrowClassName })
+    let dropdownArrow = dropdownArrowFactory({
+      wrapperClassName: dropdownArrowWrapperClassName,
```

Comment:
**issue** I think this is a breaking change. The current `dropdownArrow` option only expects a `className`, meaning existing implementations will ignore the new `wrapperClassName` and potentially show an arrow when they're not meant to.